.github/workflows/docs.yml

```diff
@@ -0,0 +1,48 @@
+name: Deploy Documentation
+
+on:
+  workflow_dispatch:
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
I like the inclusion of this one. I would recommend setting it to an exact version (also for the other actions) to avoid breaking changes in a minor update (happens with super-linter f.e. all the time). For this action it would be 1.0.15. An updated update could be done via Dependabot.

Reply:
Thanks! Fixed in [ac467fa](https://github.com/unitycatalog/unitycatalog/pull/677/commits/ac467fa5cf442e7150090b1d4a3e019369395b2e)